runtimebp/internal/maxprocs/maxprocs.go

```diff
@@ -74,7 +74,7 @@ var (
 		ubermaxprocs.Set(
 			ubermaxprocs.Min(2),
 			ubermaxprocs.Logger(func(s string, i ...interface{}) {
-				fmt.Fprintf(os.Stderr, s, i...) // contains context
+				fmt.Fprintf(os.Stderr, s+"\n", i...) // contains context
```

Comment:
Another concern of not using a real logging library is that this could get interleaved, but (a) this happens early in setup, so there shouldn't be any other goroutines running and (b) these log messages are small, should always be smaller than PIPE_BUF, so the OS guarantees they'll be atomic.